vizro-core/tests/unit/vizro/actions/test_parameter_action.py

```diff
@@ -341,9 +341,9 @@ def test_one_parameter_per_target_multiple_attributes(
 
         # Run action by picking the above added action function and executing it with ()
         result = model_manager[f"{PARAMETER_ACTION_PREFIX}_test_parameter_scatter"].function()
-        assert result["scatter_chart"] == target_scatter_parameter_y_and_x
-        assert "box_chart" not in target_scatter_parameter_y_and_x
+        assert result.scatter_chart == target_scatter_parameter_y_and_x
```

Comment:
(Comment applies in many places but is not very important if you think it's not worth doing)

Reply:
Totally makes sense. Since namedtuple is order sensitive, I added `.as_dict()` on `result` and `expected` variables so we can change the order of the targets in the source code and still keep tests passing.

Reply:
Perfect, thank you!